### PR TITLE
Adding pyhdbpp, python extractor for tango-hdb++ databases

### PR DESCRIPTION
**Pull request by sergirubio**
pyhdbpp is a python module for extracting data from Tango Control System archiving databases (HDB++), it actually supports mysql/mariadb and timescaledb. It is used by some other packages already available in conda-forge (taurus).

@conda-forge/help-python

### CODE REVIEW

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/pyhdbpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  commands:
+    - pip check
+    - hdbpp-reader --help
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/tango-controls/hdbpp/libhdbpp-python
+  summary: hdb++ python3 API
+  license: LGPL-3.0
```
[the remark this answers is not in the document]
Comment (sergirubio) [reply recipes/pyhdbpp/meta.yaml]: thanks for the comment, I updated the license to LGPL-3.0-or-later

File: recipes/pyhdbpp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% set name = "pyhdbpp" %}
+{% set version = "1.5.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyhdbpp-{{ version }}.tar.gz
+  sha256: 41b6a82cfdc64aa9770aa1c3e4d05edd767537012a4a6765c633b01fbe1f3b02
```
[the remark this answers is not in the document]
Comment (sergirubio) [reply recipes/pyhdbpp/meta.yaml]: hash updated, thanks for the comment

**Comment by beenje** [comment]
You still have a script in your package that makes the build fail on windows: `[noarch_python] Noarch package contains binary script: hdb2csv.exe`

Could you convert it to an entry-point?

**Comment by beenje** [comment]
I think you just have to add the entry point to the conda recipe as well

**Pull request merged**